src/util/get-files.ts

```diff
@@ -409,7 +447,8 @@ async function explode(paths, { accepts, output }) {
     if (s.isDirectory()) {
       const all = await fs.promises.readdir(file);
       /* eslint-disable no-use-before-define */
-      return many(all.map(subdir => asAbsolute(subdir, file)));
+      const recusive = many(all.map(subdir => asAbsolute(subdir, file)));
+      return recusive as any as Promise<string | null>;
```

Comment:
TypeScript can't handle recursive types 😖 

So I have to type cast to "fake-it-til-i-make-it" 😆 

See this issue: https://github.com/Microsoft/TypeScript/issues/28929

Reply:
`recusive` -> `recursive`

Reply:
Fixed 👍